homeassistant/components/nws/strings.json

```
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "title": "National Weather Service (NWS)",
```

I just merged a PR that moves "title" into the root of this object.
```suggestion
  "title": "National Weather Service (NWS)",
  "config": {
```